src/encoded/static/components/datacolors.js

```diff
@@ -221,7 +221,14 @@ class DataColors {
     //         integer -- do not pass a string with a "%" sign at the end.
     colorList(keys, options) {
         let colors = [];
-        if (keys && keys.length > 0) {
+        if (options && keys && options.merryGoRoundColors && keys.length > 0) {
```

Comment:
DataColors is a module to be used by anyone, so could you document what `merryGoRoundColors` is for, the way `tint` is documented?